Error: Cannot find module '../../babel.config.js'

1. I cloned a fresh repo.
2. Ran `yarn`.
3. Ran `yarn example:react`.

I get the following error:

```
$ yarn example:react                                                                   master
$ polkadot-dev-build-ts
$ yarn polkadot-dev-clean-build
$ polkadot-dev-clean-build
$ yarn polkadot-exec-tsc --emitDeclarationOnly --outdir ../build
$ tsc --emitDeclarationOnly --outdir ../build
*** @polkadot/example-react 0.80.2-1
Successfully compiled 2 files with Babel (282ms).
Successfully compiled 2 files with Babel (38ms).

*** @polkadot/example-vue 0.80.2-1
Successfully compiled 2 files with Babel (59ms).
Successfully compiled 2 files with Babel (17ms).

*** @polkadot/react-identicon 0.80.2-1
Successfully compiled 12 files with Babel (115ms).
Successfully compiled 12 files with Babel (56ms).

*** @polkadot/react-qr 0.80.2-1
Successfully compiled 13 files with Babel (108ms).
Successfully compiled 13 files with Babel (57ms).

*** @polkadot/reactnative-identicon 0.80.2-1
Successfully compiled 8 files with Babel (34ms).
Successfully compiled 8 files with Babel (20ms).

*** @polkadot/ui-assets 0.80.2-1
Successfully compiled 3 files with Babel (5ms).
Successfully compiled 3 files with Babel (3ms).

*** @polkadot/ui-keyring 0.80.2-1
Successfully compiled 22 files with Babel (153ms).
Successfully compiled 22 files with Babel (103ms).

*** @polkadot/ui-settings 0.80.2-1
Successfully compiled 12 files with Babel (73ms).
Successfully compiled 12 files with Babel (45ms).

*** @polkadot/ui-shared 0.80.2-1
Successfully compiled 28 files with Babel (95ms).
Successfully compiled 28 files with Babel (62ms).

*** @polkadot/vue-identicon 0.80.2-1
Successfully compiled 10 files with Babel (61ms).
Successfully compiled 10 files with Babel (17ms).

/tmp/ui/node_modules/webpack-cli/bin/cli.js:93
                                throw err;
                                ^

Error: Cannot find module '../../babel.config.js'
Require stack:
- /tmp/ui/packages/example-react/webpack.config.js
- /tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js
- /tmp/ui/node_modules/webpack-cli/bin/cli.js
- /tmp/ui/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (/tmp/ui/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/tmp/ui/packages/example-react/webpack.config.js:20:22)
    at Module._compile (/tmp/ui/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (/tmp/ui/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at WEBPACK_OPTIONS (/tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js:114:13)
    at requireConfig (/tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js:116:6)
    at /tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js:123:17
    at Array.forEach (<anonymous>)
    at module.exports (/tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js:121:15)
    at /tmp/ui/node_modules/webpack-cli/bin/cli.js:71:45
    at Object.parse (/tmp/ui/node_modules/webpack-cli/node_modules/yargs/yargs.js:576:18)
    at /tmp/ui/node_modules/webpack-cli/bin/cli.js:49:8
    at Object.<anonymous> (/tmp/ui/node_modules/webpack-cli/bin/cli.js:366:3)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/tmp/ui/node_modules/webpack/bin/webpack.js:156:2)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/tmp/ui/packages/example-react/webpack.config.js',
    '/tmp/ui/node_modules/webpack-cli/bin/utils/convert-argv.js',
    '/tmp/ui/node_modules/webpack-cli/bin/cli.js',
    '/tmp/ui/node_modules/webpack/bin/webpack.js'
  ]
}
```

Version:
```
$ node --version                                                                    
v14.17.0

$ yarn --version                                                                     
3.0.0-rc.2
```

--- a/packages/example-react/webpack.config.js
+++ b/packages/example-react/webpack.config.js
@@ -17,7 +17,7 @@ module.exports = {
         use: [
           {
             loader: 'babel-loader',
-            options: require('../../babel.config.js')
+            options: require('../../babel.config.cjs')
           }
         ]
       }

--- a/packages/example-vue/webpack.config.js
+++ b/packages/example-vue/webpack.config.js
@@ -18,7 +18,7 @@ module.exports = {
         use: [
           {
             loader: 'babel-loader',
-            options: require('../../babel.config.js')
+            options: require('../../babel.config.cjs')
           }
         ]
       },

